tests/background/sidebar-injector-test.js

```diff
@@ -208,6 +255,70 @@ describe('SidebarInjector', function () {
       });
     });
 
+    describe('when viewing a VitalSource book', () => {
```

Comment:
This whole series of tests could be really complicated, but they're clear and easy to understand. Good job.